JPS_Ontology/ontology/ontoapplication/OntoApplication.csv

```diff
@@ -0,0 +1,21 @@
+OntoApplication,TBox,https://www.theworldavatar.com/kg/ontoapplication/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoApplication,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoApplication,TBox,OntoApplication is an ontology for representing software applications and their services.,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoApplication,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+Application,Class,,,,,,A computer program designed to help people perform an activity.,https://www.theworldavatar.com/kg/ontoapplication/,Application
+MobileApplication,Class,Application,IS-A,,,,A software application designed to run on a mobile device.,https://www.theworldavatar.com/kg/ontoapplication/,Mobile application
+Inbox,Class,,,,,,An e-folder for receiving and storing messages.,https://www.theworldavatar.com/kg/ontoapplication/,Inbox
+Message,Class,,,,,,"A communication containing some information, news, advice, request, or the like.",https://www.theworldavatar.com/kg/ontoapplication/,Message
+MessageCategory,Class,,,,,,The category in which a message information falls under.,https://www.theworldavatar.com/kg/ontoapplication/,Message category
+AssetInformation,Class,MessageCategory,IS-A,,,,A type of message pertaining to asset information.,https://www.theworldavatar.com/kg/ontoapplication/,Asset information
+MaintenanceInformation,Class,MessageCategory,IS-A,,,,A type of message pertaining to maintenance information.,https://www.theworldavatar.com/kg/ontoapplication/,Maintenance information
+hasInstalled,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/SmartPhone,MobileApplication,,A relationship that describes the mobile application installed on a smart phone.,https://www.theworldavatar.com/kg/ontoapplication/,has installed
+hasInbox,Object Property,,,Application,Inbox,,A relationship that describes the inbox provided by an application.,https://www.theworldavatar.com/kg/ontoapplication/,has inbox
+contains,Object Property,,,Inbox,Message,,A relationship that describes the messages contained in an inbox.,https://www.theworldavatar.com/kg/ontoapplication/,contains
+hasMessageCategory,Object Property,,,Message,MessageCategory,,A relationship that describes the category in which a message falls under.,https://www.theworldavatar.com/kg/ontoapplication/,has message category
+hasMessageID,Data Property,,,Message,String,,An identifier assigned to a message.,https://www.theworldavatar.com/kg/ontoapplication/,has message ID
+hasTitle,Data Property,,,Message,String,,A descriptive or general heading of a message.,https://www.theworldavatar.com/kg/ontoapplication/,has title
+hasContent,Data Property,,,Message,String,,The information contained in a message.,https://www.theworldavatar.com/kg/ontoapplication/,has content
+hasFirebaseID,Data Property,,,Application,String,,The firebase Id assigned to an application.,https://www.theworldavatar.com/kg/ontoapplication/,has firebase ID
```

Comment:
This piece is not aligned with the MIRO board. Shall `applicationID` and `firebaseID` only apply to MobileApplications or general Applications? I believe the 2nd makes more sense (and is the currently encoded structure); however, MIRO suggests differently

Reply:
Same applies to `hasInbox`

Reply:
I was thinking whether it will be better to have applicationID, firebaseID and hasInbox more general because it can apply to other types of software applications and not just mobile applications.